lua/sol.lua

```diff
@@ -0,0 +1,21 @@
+return function (net_conf, gk_conf, lcore)
+
+	-- Init the Solicitor configuration structure.
+	local sol_conf = gatekeeper.c.alloc_sol_conf()
+	if sol_conf == nil then
+		error("Failed to allocate sol_conf")
+	end
+
+	sol_conf.lcore_id = lcore
+	sol_conf.pri_req_max_len = 1024
+	sol_conf.enq_burst_size = 1024
+	sol_conf.deq_burst_size = 1024
```

Comment:
Don't quite understand this comment -- are you saying that `enq_burst_size` and `deq_burst_size` should be removed and we should just use `GATEKEEPER_MAX_PKT_BURST`? Or that the code should make sure they are equal? Or something else?